azure/pipelines/templates/deploy.yml

```diff
@@ -192,7 +192,9 @@ jobs:
                 -govukNotifyCallbackAPIKey "${{parameters.govukNotifyCallbackAPIKey}}"
                 -sandbox "${{parameters.sandbox}}"
                 -googleAnalyticsApply "${{parameters.googleAnalyticsApply}}"
-                -googleAnalyticsManage "${{parameters.googleAnalyticsManage}}"'
+                -googleAnalyticsManage "${{parameters.googleAnalyticsManage}}"
+                -googleTagManagerApply "${{parameters.googleTagManagerApply}}"
```

Comment:
we are setting `googleAnalyticsApply` and `googleAnalyticsManage` earlier in the deploy template: https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/azure/pipelines/templates/deploy.yml#L69 should we do the same for `googleTagManagerApply` and `googleTagManagerManage`?

Reply:
Should we also add those in the `release.yml` for prod? https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/azure/pipelines/release.yml#L207

Reply:
:+1: The docs might be a bit thin on detail here https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/docs/environment-variables.md but I'll do what has been done for analytics to be sure.